modules/graphstudio/pages/build-graph-patterns/visual-query-builder-overview.adoc

```diff
@@ -1,14 +1,14 @@
 = Visual Query Builder
 
-The Visual Query Builder (VQB) provides a visual approach to building your graph business logic. In the Build Graph Patterns page, you can create visual patterns in a drag-and-drop fashion to represent the questions you want to ask instead of writing GSQL queries.
-
-By adding the Visual Query Builder component, GraphStudio becomes a complete visual SDK for users to _build graph applications from end to end without writing a single line of code_!
+The Visual Query Builder (VQB), found on the Build Graph Patterns page, provides a visual approach to exploring your graph and using it to solve problems.
+Here, you can create visual patterns by dragging and dropping vertices and edges to represent the questions you want to ask instead of writing GSQL queries.
 
 image::vqb-overview.png[]
 
 == Basic Concepts
 
-A visual pattern is a declarative way of describing a template subgraph structure, and all the subgraph structures that the visual pattern is *homomorphic* with are valid matching results.
+When you create a graph structure, you are creating a subgraph representing part of the full graph.
```

Comment:
create a visual pattern? i think visual pattern is the term we are using in the product?